<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1031</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>392</x>
        <y>200</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Eleve</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>392</x>
        <y>280</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>activite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>400</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="stat">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>341</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="View">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>431</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeRecharche">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="archive">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>archuve</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>530</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>inscription a une activite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mail">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>530</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>530</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>export</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>retour</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
